clang-tools-extra/include-cleaner/tool/IncludeCleaner.cpp

```diff
@@ -173,6 +173,7 @@ class Action : public clang::ASTFrontendAction {
     if (!HTMLReportPath.empty())
       writeHTML();
 
+    // Source File's path relative to compilation database.
     llvm::StringRef Path =
```

Comment:
can we keep reporting with abs-paths here, i.e:
```
llvm::SmallString<256> AbsPath(SM.getFileEntryRefForID(SM.getMainFileID())->getName());
SM.getFileManager().makeAbsolutePath(AbsPath);
...
```

This way we can recognize these paths in main and perform the mapping accordingly.